corehq/apps/app_manager/templatetags/app_manager_extras.py

```diff
@@ -16,3 +16,9 @@ def get_available_modules_for_case_list_configuration(app, module):
             and m.module_type not in disallowed_module_types
             and m.case_type == module.case_type)
     ]
+
+
+@register.filter
+def get_available_modules_for_case_tile_configuration(app, module):
```

Comment:
if you read the function as a sentence "get available modules for case tile configuration [for] app [but exclude this] module"

I don't know not really that important